scripts/sign-release-artifacts.sh

```
@@ -0,0 +1,98 @@
+  echo "Signing: $file"
+
+  cosign sign-blob "$file" \
+    --bundle "$file.bundle" \
+    --output-signature "$file.sig" \
+    --output-certificate "$file.pem"
+done
+
+echo "Generating SHA256SUMS.txt"
+(
+  cd "$OUTPUT_DIR"
+  shasum -a 256 * > SHA256SUMS.txt
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)